### ISSUE TITLE
Function name must be a string

### ISSUE THREAD

**Issue by xguntis** [weidizhang/smart-php-obfuscator#1]
It failed to de-obfuscate, indeed, tub the resulting code gives me error:

Fatal error: Uncaught Error: Function name must be a string in /.../.../filename.php(2) : eval()'d code:1 Stack trace: #0

**Comment by Demodian** [comment]
Yeah, this one is happening to me as well.  This library had promise for being lightweight, but it doesn't work when it needs to.
